docusaurus/docs/iOS/02-tutorials/03-livestream.mdx

```diff
@@ -5,328 +5,325 @@ description: How to build a livestream experience using Stream's video SDKs
+To create the first call object, specify the call type as **livestream** and provide a unique **callId**. The **livestream** call type comes with default settings that are usually suitable for livestreams, but you can customize features, permissions, and settings in the dashboard. Additionally, the dashboard allows you to create new call types as required.
+
+Finally, using `call.join(create: true)` will not only create the call object on our servers but also initiate the real-time transport for audio and video. This allows for seamless and immediate engagement in the livestream.
+
+Note that you can also add members to a call and assign them different roles. For more information, see the [call creation docs](../03-guides/02-joining-creating-calls.mdx).
 
-Next, let's see the contents of the `LivestreamHostView`. Note that some UI code and modifiers are removed to focus on the most important parts (please refer to our repo for the full implementation):
+### Step 4 - Rendering the video
+
+In this step we're going to build a UI for showing your local video with a button to start the livestream. This example uses SwiftUI, but you could also use UIKit.
+
+In `LivestreamSampleApp.swift` replace the `LivestreamView` with the following code:
 
 ```swift
-var body: some View {
-    VStack {
-        HStack {
-            Text("Live")
-                .opacity(isBroadcasting == true ? 1 : 0)
-        }
-        
-        GeometryReader { reader in
-            if let first = participants.first {
-                VideoCallParticipantView(
-                    participant: first,
-                    availableSize: reader.size,
-                    contentMode: .scaleAspectFit,
-                    edgesIgnoringSafeArea: .bottom
-                ) { participant, view in
-                        view.handleViewRendering(for: participant) { size, participant in }
+struct LivestreamView: View {
+    
+    @Injected(\.streamVideo) var streamVideo
+    
+    let call: Call
+    
+    @StateObject var state: CallState
+    
+    init(call: Call) {
+        self.call = call
+        _state = StateObject(wrappedValue: call.state)
+    }
+    
+    var body: some View {
+        VStack {
+            HStack {
+                Spacer()
+                
+                Text("Live \(state.participantCount)")
+                    .bold()
+                    .padding(.all, 4)
+                    .padding(.horizontal, 2)
+                    .foregroundColor(.white)
+                    .background(Color.blue)
+                    .cornerRadius(8)
+                    .opacity(call.state.backstage ? 0 : 1)
+                    .padding(.horizontal)
+            }
+            
+            GeometryReader { reader in
+                if let first = state.participants.first {
+                    VideoRendererView(id: first.id, size: reader.size) { renderer in
```

Comment:
Why not the higher level `VideoCallParticipantView`?

Reply:
To be consistent with Android - for this use-case it doesn't make any difference.